Source/Install/MediaInfo_GUI_Windows.nsi

```diff
@@ -210,6 +222,16 @@ Section Uninstall
     ExecWait '"$INSTDIR\ffmpeg_plugin_uninst.exe" /S _?=$INSTDIR'
     Delete "$INSTDIR\ffmpeg_plugin_uninst.exe"
 
+  ${If} ${AtLeastWin11}
+    System::Call '"$INSTDIR\MediaInfo_PackageHelper.dll"::_Uninstall@0() ? u'

```

Comment:
We could do that too (also install) when the user change the preferences, it seems easy and no interaction at all if the user does not want the shell extension.

Reply:
try removing a package from within an app that belongs to the package, deadlock of app waiting for uninstall to complete while Windows Installer waits for app to exit
also removing the package will make MediaInfo no longer have package name in Task Manager
and we are not supposed to install and uninstall MSIX packages just for enabling/disabling a shell extension, the package represents the entire app